en/docs/integrations/build-your-first-integration.md

```diff
@@ -142,6 +142,8 @@ Now you have successfully created and configured the integration. It looks as fo
 
 ## Step 6: Try out the integration
 
+To try out your integration, follow this procedure:
```

Comment:
follow the steps below?

Reply:
The style guide recommends to avoid the use of directional language such as above, below, or right-hand side[1]. Therefore, better to avoid those terms I suppose.
[1] use directional language to orient the reader, such as above, below, or right-hand side.